plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/part/EntityPart.java

```diff
@@ -97,6 +97,17 @@ public void performRequest(Request request) {
         }
     }
 
+    /**
+     * VQB isn't supposed to have column relations, so
+     * getModelSourceConnections and getModelTargetConnections should return
+     * only entity connections
+     *
+     * @return is this part VQB related
+     */
+    protected boolean isVQB() {
```

Comment:
Perhaps it should be called `isHideAttributeAssociations` and the default implementation will perform checks for `ERDUIConstants.PREF_ROUTING_TYPE` and `ERDAttributeVisibility.isHideAttributeAssociations`?

I don't think the ERD should know about the VQB in that particular context.